src/main/scala/cc/factorie/app/nlp/lexicon/ResourceLexicons.scala

```diff
@@ -28,11 +28,13 @@ class ResourceLexicons(val sourceFactory: String=>io.Source, val tokenizer:Strin
   class ChainWordLexicon(name:String)(implicit dir:String) extends cc.factorie.app.nlp.lexicon.ChainWordLexicon(dir+"/"+name, tokenizer, lemmatizer) {
     this ++= sourceFactory(dir + "/" + name + ".txt")
   }
+
   /** deprecated **/
```

Comment:
I was just replicating the comment Kate put above the ChainPhraseLexicon on the PhraseLexicon. I can't think of a reason to use the PhraseLexicon, but the TriePhraseLexicon and ChainPhraseLexicon have different token annotation methods (though the ChainPhraseLexicon is much slower at annotation time).